ppml/trusted-big-data-ml/python/docker-graphene/start-scripts/start-python-fl-server-sgx.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+cd /ppml/trusted-big-data-ml
+/graphene/Tools/argv_serializer bash -c " /opt/jdk8/bin/java\
+        -cp '/ppml/trusted-big-data-ml/work/spark-3.1.2/conf/:/ppml/trusted-big-data-ml/work/spark-3.1.2/jars/*'\
+        -Xmx10g org.apache.spark.deploy.SparkSubmit\
+        --master 'local[4]'\
+        /ppml/trusted-big-data-ml/work/examples/start-fl-server.py -p 8981 --client-num=3" > /ppml/trusted-big-data-ml/secured-argvs
```

Comment:
is client-num determined when server starts?

Reply:
better to pass args for port and client-num

Reply:
> is client-num determined when server starts?

Yes, server must know the client number before starting